crawler_message_item_test.go

```diff
@@ -38,21 +39,24 @@ var _ = Describe("CrawlerMessageItem", func() {
 			Path:   "/government/organisations",
 		}
 
-		rootURL = &url.URL{
-			Scheme: "https",
-			Host:   "www.gov.uk",
-			Path:   "/",
-		}
-
 		delivery = amqp.Delivery{Body: []byte(testURL.String())}
-		item = NewCrawlerMessageItem(delivery, rootURL, []string{})
+		item = NewCrawlerMessageItem(delivery, rootURLs, []string{})
+
+		urlA, _ = url.Parse("https://www.gov.uk")
```

Comment:
Given the work that @mattbostock did in #89 it would be nice to be consistent and avoid `url.Parse` and instead instantiate URLs using `&url.URL{...}`. But, it's a nice to have and doesn't block this from being merged.